.travis.yml

```
@@ -1,25 +1,46 @@
+    - secure: Yx/hWeG2EZV25A5015Y9Onnv5oZvamEZeISQ9OtmJ0nsjJBq6uRZUI0YxWiaeNRzI/7uRpbE45BMMiWZmIZlT1CXfOKyMIPouJFDiXEEEipgX2K3yuj0LCaP6ghUN0gZdSSP8eCOAlZkJ0G0IeqqjmIQixhld/9HzLjcaJZ+zp9gULI6UkEtX/A1T+0llhK6qONVJlDIvjyyBQWePGpuAsKl4DFui9UZO/M7rE2c6sgCn2S6LbfVJUcJkpblP7RPlIukpiFKgsajvqnZNwPltepE0W136NJ8IlNHgh56oraqgTMLvAyuYn6wAIxb0elpDTxjlxCfflrU8O5U1tyM0MLlZgtXYVCXuIn12H49rwYtC77ooK7IHhzSoIBtbEibX9FSdGTKitDDP+W+8SPjipeQfcVahTJLs9qWk869ASYAxiKvOWP3UnySbBGXTMOvFWLQIuy3fwpDaNryy45d8RQqmwztZ0ikaqyi+3rRGJZQnaXO52+wLBjNdeBt2XW29yzM8Vl7Jxqg4WbatNcRt6R7v9+pkxkSParo+1fhPu3NMZyIy/q8ZxI2e2Q2T//hdZVCvEUeY31cULG5ugRnOUSvUpUKp/+DUH90bSfS8TJ1uW1Ph8VS711rl59ucb1RpHi4VN2gDJEKUXRtFJj8XohugKnrXJrdirMaF8w8/78=
+    - secure: "ln4rP+gtJ8UxeH4kJ6Ns1JQ+HcsEowDBlGO3B/zMoA3ZnNMIS9vU6aOg78ikeowgMvQFzUowMJtsd0iEsUTzs13j22/UEaInAFd/slFmFhLPmntVviP/kh95agoWA2oeNYYBDDTnHwKstu3FM+moerzpVaNfRn821W5VJhK16MDx/YbHVI53wFslKk/PXfjwAzAnZIN1+RnSzViKl0DwQs0iABAbZfXQKtxE9NyqZkNR5a4QteT+B0Lxn5iC67oTkjk7MKhBbEO9D9Zm5tuwRZjtWuFyZg0BoItz2v7xt6eshz1JDvzny/bGhISaXU3eBdxz12Zy0YNx4eOTzmWaU9A+Yfs4yBmYrESnuJmWND6kxuxeD9ynZDpgoccqQ3yx5PFu6io0ITlxLod0e3iXx3KDA9+nBVm+u2InHvUv6A90Qk6H8QgW37FjWnpYQnXG2N6idC5OzJZEUzk0Q6yceZ9O57z0onbL3aXGrgbu9RICBOCfT00gpwG+it5rC7VynTwKk+Z2aVP1WcroK6eyDVFeExsxzWAxK70HGsPDd7pjNtWPg8uAc5Qc3+SXcbpypw3GczSnzWrhVGKgoIasBXNfVMJH6j8cpINN5zorzxv97wmxzjbQPBRjDH0RkK3gqQUYTQFgifnoThxl/6yGoYdneDJlWpQOLLECyyqGWPk="
+
+matrix:
+  include:
+    - language: python
+      python: 3.8
+      script: "./bin/ci"
+      deploy:
+        - provider: pypi
+          user: dev-im
+          edge:
+            source: ssbarnea/dpl
+            branch: master
+          password:
+            secure: hhjn47GalCnADFUsLerOZALSHISeJbXyZ3N2UzChGsO3y3mERY7TD5eGFgXSkr+KlOq3Z96Im+N2f4TviRo9F87i0Z1jYAIIURDYFTJ90UQUugdyY5hjuPDDUYbEZbkcZVroW3Uq9dUpe4KcAnVFZmwIT0DEUTEgdyNzgkxMNBEobu/ZAu2tkWHu6PTWaJcpp+jII3vpay1o3HJbGdrbX9DLyC9ODHG7XJuePddQmDLuOTKOfnNxqo1tBkaw3a49bW2G1JWYpfof7l8AMaeCikRownAd0NrolDkRc19VfflvSe4w1FMYRGnZy1IJRo16olQEQiyDp8vBbAFe0xq6yncUsf12wXyRGFQqDgJXJImm0lg3oj8/4ag+ZxR8XvxGEgb+IWuqy+CD9bAVBbKRWQKeCgg2PQjTl7INoDIAaJ8f/Z65fhSB3uPZ1FlE8pBnNSFF0Lma2HGp1fIFM+jEH+WLbZKrUvYTXYvbfTUMwY41+u6Q+PEEkhqvNslOFTn2TZXwJSk0Eyq/FfhGS4jw5Bs/ngBybVSoB4ypHClo/dJcAVCl2KDx7Pc+GI1ZxNE1D6sJnOS4E6jJJdZYC9s/UQacHvTke+G3dXii9Y9BzEd508irw2DC0s+uJnSGzUm57tWHEbNAn+noZ0dhzLwCl7Jb6Uqe5sZDHK5/CqUcZeE=
+          on:
+            tags: true
+
+    - language: node_js
+      node_js: 12
+      script:
+        - cd hmt_escrow_js
+        - yarn install
+        - npm run copy-contracts
+        - ./bin/ci
+      deploy:
+        - provider: npm
+          edge: true # opt in to dpl v2
+          access: public
+          tag: alpha
+          on:
+            branch: npm-publish-js
+
```

```suggestion
          branch: master
          on:
            tags: true
```
@gaieges @excerebrose We should publish only on tag release from master, like here: https://github.com/hCaptcha/hmt-escrow/blob/3fdadc802487e1c1a56eb011fa37866f5546d0ff/.travis.yml#L20-L30

Otherwise, it's gonna publish our package every time we commit to this branch, and we have to change the version every other commit (that's why the latest build fails).

---

I'll flip it back to master but not tying it to tags yet, as the python package is already versioned and they wont move together for now